sdk/src/azure/core/az_hfsm.c

```diff
@@ -20,6 +20,8 @@
 #include <azure/core/internal/az_result_internal.h>
 #include <azure/core/internal/az_precondition_internal.h>
 
+#include <azure/core/_az_cfg.h>
+
 const az_hfsm_event az_hfsm_event_entry = { AZ_HFSM_EVENT_ENTRY, NULL };
```

Comment:
Note: I believe applications will require access to all event types for logging purposes.